extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -381,11 +387,9 @@ public function getPage() {
 						$data['article'] = $articleData;
 
 						if ( !$title->isContentPage() ) {
-							// This does two things:
-							// - gets displayTitle from parser function {{DISPLAYTITLE:displayTitle}}
-							// - removes the namespace prefix from it
-							$data['article']['displayTitle'] =
-								Title::newFromText( $data['article']['displayTitle'] )->getText();
+							// Remove the namespace prefix from display title
+							$displayTitle = Title::newFromText( $displayTitle )->getText();
```

Comment:
This can throw on you
https://github.com/Wikia/app/blob/dev/includes/Title.php#L135